### PR TITLE
[20] Add trainee show page placeholder

### PR DESCRIPTION
**Pull request by asmega**
### Context

- https://trello.com/c/QH5tHvV7/20-s-create-trainee-show-page
- Add placeholder trainee show page so we can hang stuff off it

### Changes proposed in this pull request

- Add placeholder controller and views to show a trainee
- Bump `.nvmrc` to match version in `package.json`

### Guidance to review

- visit http://localhost:5000/trainees/1 - NOTE change to correct port
- should see trainee page

### CODE REVIEW

**Review by gpeng — APPROVED** [review]

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-12.13.1
+12.18.3
```
Comment (defong) [inline .nvmrc:1]: probably need to update readme...

**Review by defong — APPROVED** [review]

File: .nvmrc (hunk 1 above)
Comment (asmega) [reply .nvmrc:1]: I'll probably delete it from the readme

**Comment by aliuk2012** [comment]
Why is do we have a new empty file called `app/webpacker/packs/application.css`?

**Comment by asmega** [comment]
> Why is do we have a new empty file called `app/webpacker/packs/application.css`?

It was throwing an error for me so I just added the file. Is there a better way of doing this?

**Comment by aliuk2012** [comment]
We should be able to delete `app/webpacker/packs/application.css` without there any any issues. I tried this locally and it worked for me with out having this file available.

**Pull request merged**